README.md

```diff
@@ -78,6 +78,8 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Fork the Repository.
   * Clone the repository to your local machine `$ git clone https://github.com/<your-github-username>/JagratiWebApp.git`
   * Change directory to JagratiWebApp `$ cd JagratiWebApp`
+  * Add a reference to the original repository. <br>
+   `$ git remote add upstream https://github.com/garg3133/JagratiWebApp.git`
```

Comment:
Putted a line break here.

Reply:
Don't but a line break here. Put two spaces after the sentence, it works as a line break